lib/validate.js

```diff
@@ -81,6 +120,29 @@ module.exports = function validateMiddleware (opts) {
               msg: 'Bad Request - ' + err.toString()
             });
           } else {
+            // Assemble object from properties deemed valid from
+            // all Joi schema validations.
+            var validParams = _.assign.apply(_, joiedParams);
```

Comment:
Do we need this _assign_. I might be missing something, could we not use _joiedParams_ directly?